docs/docs/pricing/README.md

```diff
@@ -31,6 +31,18 @@ Free Tier accounts have a [daily credit limit](/limits#daily-credits-limit) and
 
 Upgrading to a [paid tier](https://pipedream.com/pricing) will increase the number of available active workflows and connected accounts.
 
+#### Compute Time Limits
+
+Additionally Free accounts have a limit on compute time for processing test events in the workflow builder.
```

Comment:
```suggestion
Additionally, free accounts have a limit on compute time for processing test events in the workflow builder.
```

Reply:
Changed "Free" to "free" and added a comma after "Additionally" (Google's Developer docs style guide)